assets/chat/js/emitter.js

```diff
@@ -20,6 +20,18 @@ class EventEmitter {
     }
     return false;
   }
+
+  off(name, fn) {
```

Comment:
Since you've fixed the duplicate message issue in commit https://github.com/destinygg/chat-gui/pull/711/commits/34e6721ea39ec16d2dfe7e12769c291ce15af1ab this doesn't seem to be used anywhere, prob can just remove it.